float64 bearing		# Horizontal bearing relative to sensor frame
float64 distance
float64 angle		# Vertical angle deviation from vertical
